manuscripts2/new_functions.py

```diff
@@ -0,0 +1,510 @@
+import logging
+
+from collections import OrderedDict, defaultdict
+from datetime import timezone
+from dateutil import parser
+
+import pandas as pd
+from elasticsearch import Elasticsearch
+from elasticsearch_dsl import A, Q, Search
+
+logger = logging.getLogger(__name__)
+
+
+class Index():
+    """Index class which will represent an index in elasticsearch"""
+
+    def __init__(self, index, url=None):
+        """
+        :url: the address/url of the elasticsearch instance to be used?
```

Comment:
Respect the order of the arguments, whatever you use in the list of parameters to the function (in this case, I think 'index' would be the first one).

Besides, the correct way of writing it is 

```
  :param url: comment
```

(note "param")

We're using Sphynx conventions, see Sphynx entry in [Formats](https://stackoverflow.com/a/24385103/2075265)